lib/colorado_lottery.rb

```diff
@@ -0,0 +1,46 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = Hash.new { |h, k| h[k] = [] }
+    @winners = []
+    @current_contestants = Hash.new { |h, k| h[k] = [] }
+  end
+
+  def interested_and_18?(contestant, game)
+    return true if contestant.game_interests.include?(game.name) && contestant.age >= 18
```

Comment:
Unnecessary guard statement here. We just need the condition for any and all boolean methods: 
`contestant.game_interests.include?(game.name) && contestant.age >= 18`